drivers/counter/Kconfig.nrfx

```diff
@@ -112,6 +112,16 @@ config COUNTER_RTC0_PRESCALER
 	help
 	  Frequency = 32768 / (prescaler+1).
 
+config COUNTER_RTC0_PPI_WRAP
```

Comment:
I would like to do kconfig/DTS changes as separate PR with no functional changes. That will be easier to review. Since there are already multiple options that should be moved - instance enablling, prescaler, ppi_wrap.

Reply:
OK.